sys-kernel/gentoo-kernel/gentoo-kernel-5.10.224.ebuild

```
@@ -77,14 +77,14 @@ src_prepare() {
 			cp "${DISTDIR}/kernel-x86_64-fedora.config.${CONFIG_VER}" .config || die
 			;;
 		arm)
-			return
+			touch .config || die
```

```suggestion
			> .config || die
```

Also maybe merge them into a single block.